config/application.rb

```diff
@@ -31,5 +31,7 @@ class Application < Rails::Application
     config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
 
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{yml}')]
+
+    routes.default_url_options[:host] = Figaro.env.mailer_domain_name
```

Comment:
I noticed that for development we have it set to `domain_name: 'localhost:3000'` (no HTTP). for prod, it is `'login.gov'` whereas `mailer_domain_name ` is `'https://login.gov'`

should we create a separate figaro key for this use case? I think using `mailer_domain_name` outside of a mailer is kind of confusing...but `domain_name` will not work as-is